docs/spring-boot/app-starter.md

```diff
@@ -116,30 +116,95 @@ public class ExtAppSpringApplication {
 By configuring the property `bdk.app.auth.enabled=true`, the Application backend will provide Apis for performing
 the [Circle of Trust](https://developers.symphony.com/extension/docs/application-authentication) of Symphony:
 
-- Application Authenticate: `POST /bdk/v1/app/auth`:
-  
-    Parameters:
-    ```json
-     {
-        "appToken": "your-app-token"
-     }
-    ```
-
-- Validate tokens: `POST /bdk/v1/app/tokens`:
-
-    Parameters:
-    ```json
-    {
-      "appToken": "your-app-token",
-      "symphonyToken": "symphony-token" // Symphony token returned from Symphony App Frontend
-    }
-    ``` 
-  
-- Validate jwt: `POST /bdk/v1/app/tokens`
-
-    Parameters:
-    ```json
-    {
-      "jwt": "your-jwt" // Jwt signed by Symphony App Frontend
-    }
-    ``` 
+```yaml
```

Comment:
we could put it in a separate file and load via https://editor.swagger.io/?url=https://raw.githubusercontent.com/OAI/OpenAPI-Specification/master/examples/v3.0/petstore.yaml (but how do we target a specific commit... well i guess it won't change much)